src/atoms/story-elements/embed/embed.tsx

```
@@ -18,7 +18,12 @@ export const DefaultEmbed = ({ element }: StoryElementProps) => {
   const src = getIframeContent(embedData, /(?<=src=["']).*?(?=[*"'])/);
   const scrolling = getIframeContent(embedData, /(?<=scrolling=["']).*?(?=[*"'])/);
   const title = element.subtype || element.title || "";
-  return src ? <Iframe src={src} scrolling={scrolling} title={title} /> : null;
+  if (src) {
+    return <Iframe src={src} scrolling={scrolling} title={title} />;
+  } else if (embedData) {
+    return <div dangerouslySetInnerHTML={{ __html: embedData }} />;
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)